src/app/public/modules/select-field/select-field-picker.component.html

```diff
@@ -10,6 +10,11 @@
           [selectedIds]="selectedIds"
           (selectedIdsChange)="onSelectedIdsChange($event)">
           <sky-list-toolbar>
+              <sky-list-toolbar-search-actions  >
+                  <button *ngIf="showNewButton" class="sky-btn sky-btn-default sky-select-field-picker-btn-new" (click)="emitAddNewRecord()" >
+                      <i class="fa fa-plus-circle"></i>
```

Comment:
We should use the icon component, here (it's in the `@skyux/indicators` library I believe).

```
<sky-icon icon="plus-circle"></sky-icon>
```